Dockerfile

```diff
@@ -73,6 +73,10 @@ ENV RUN_PYTHON_SDK_IN_DEFAULT_ENVIRONMENT=1
 RUN pip install .[gcp,dataflow]
 COPY . .
 
+COPY --from=apache/beam_python3.9_sdk:2.52.0 /opt/apache/beam /opt/apache/beam
+ENTRYPOINT ["/opt/apache/beam/boot"]
```

Comment:
Can you comment on the behavior observed without this change?

Reply:
Yes. Before this change, the Dataflow worker will not start up properly and the job appears to be hung for a long time without useful error messages. Product team suggested this change based on documentation on using a custom container: https://cloud.google.com/dataflow/docs/guides/build-container-image#use_a_custom_base_image